src/Entities/Entity.h

```diff
@@ -530,6 +535,11 @@ class cEntity
 	eEntityType m_EntityType;
 	
 	cWorld * m_World;
+
+	/// State variables for ScheduleMoveToWorld.
+	bool m_IsWorldChangeScheduled = false;
```

Comment:
Yes, it should be initialized in the constructor. For the comment, there was a previous block of several comments there that were triple slashes, so I changed the whole block in the commit I just now pushed.